.vscode/launch.json

```diff
@@ -4,6 +4,13 @@
     // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```

Comment:
Comments aren't allowed in JSON? _wish we could_

Reply:
They are allowed in launch.json the same way they are allowed in tsconfig.json
But github doesn’t know that it’s actually .jsonc format